test/test_pdo.py

```diff
@@ -9,27 +9,28 @@
 class TestPDO(unittest.TestCase):
 
     def test_bit_mapping(self):
```

Comment:
I agree. This is the only test that I short circuited for 2 reasons:
* I anyways want to write a more elaborate PDO test once you give me your general OK
* This test case was written for the remote node PDO, and I really don't know if this use case makes any sense. I think remote node PDOs should be read only, because sending them would make us a local node and for everything else we have SDOs